test/e2e/specs/nux.test.js

```diff
@@ -18,4 +39,145 @@ describe( 'NUX', () => {
 		const secondTipText = await page.$eval( '.nux-dot-tip', ( element ) => element.innerText );
 		expect( secondTipText ).toContain( 'You’ll find more settings for your page and blocks in the sidebar.' );
 	} );
+
+	it( 'should show "Got it" once all tips have been displayed', async () => {
+		for ( let i = 1; i < NUMBER_OF_TIPS; i++ ) {
+			await page.click( '.nux-dot-tip .components-button.is-link' );
+		}
+
+		// Make sure "Got it" button appears on the last tip.
+		const gotItButton = await page.$x( '//button[contains(text(), \'Got it\')]' );
+		expect( gotItButton ).toHaveLength( 1 );
+
+		// Click the "Got it button".
+		await page.click( '.nux-dot-tip .components-button.is-link' );
+
+		// Verify no more tips are visible on the page.
+		const nuxTipElements = await page.$$( '.nux-dot-tip' );
+		expect( nuxTipElements ).toHaveLength( 0 );
+
+		// Tips should not be marked as disabled, but when the user has seen all
+		// of the available tips, they will not appear.
+		const nuxTipsLocalStorage = await getTipsLocalStorage( page );
+		expect( nuxTipsLocalStorage.areTipsEnabled ).toEqual( true );
+		expect( Object.keys( nuxTipsLocalStorage.dismissedTips ) ).toHaveLength( NUMBER_OF_TIPS );
+	} );
+
+	it( 'should dismiss tips if "disable tips" button is clicked', async () => {
```

Comment:
s/dismiss/disable/

Reply:
So I meant "dismiss" actually because I'm checking for the tips visually being dismissed as well, but now I can see it's unclear. I'll clarify this spec.